src/index.test.ts

```diff
@@ -375,4 +375,33 @@ describe('betterAjvErrors', () => {
       },
     ]);
   });
+
+  describe('const', () => {
+    it('should handle const errors', () => {
+      data = {
+        const: 2,
+      };
+      schema = {
+        type: 'object',
+        properties: {
+          const: {
+            type: 'integer',
+            const: 42,
+          },
+        },
+      };
+      ajv.validate(schema, data);
+      const errors = betterAjvErrors({ data, schema, errors: ajv.errors });
+      expect(errors).toEqual([
+        {
+          context: {
+            allowedValue: 42,
+            errorType: 'const',
+          },
+          message: "property 'const' must be equal to constant",
```

Comment:
Could you change this to `'${prop}' property must be equal to the allowed value` to be more in line with `enum`?

Reply:
I'm actually pretty new to JSON Schema.  Do you know if `const` allows the required value to be non-scalar?  I wanted to write

```ts
          message: `'${prop}' property must be equal to the allowed value '${error.params.allowedValue}'`,
```

but it occurred to me that the allowed value may not be a scalar and thus may not be appropriate to interpolate in the template string.  Since the message is human-facing, including the value in the message seems like the most helpful approach, but it should never wind up with `[object Object]` or anything like that, and arguably there's no helpful way to distinguish between `allowedValue: null` and `allowedValue: "null"`.

For now, I can leave it out.

Reply:
I think `const` can indeed be a non-scalar. So good thinking to leave it out :+1: